fe/fe-core/src/main/java/org/apache/doris/clone/TabletScheduler.java

```diff
@@ -733,7 +741,7 @@ private Tag chooseProperTag(TabletSchedCtx tabletCtx, boolean forMissingReplica)
         for (Replica replica : replicas) {
             Backend be = infoService.getBackend(replica.getBackendId());
             if (be != null && be.isScheduleAvailable() && replica.isAlive() && !replica.tooSlow()
-                    && be.isMixNode()) {
+                    && be.isMixNode() && isSuitableBe(be.getId())) {
```

Comment:
remove here.
for a tablet  on unhealth BE,  if it's version complete, then no need to migrate it.  
If it miss version,  then sched want to clone it,  then migrate it.